packages/mongodb-log-writer/package.json

```diff
@@ -0,0 +1,90 @@
+{
+  "name": "@mongodb-js/mongodb-log-writer",
```

Comment:
Are we moving it under the `@mongodb-js/` prefix? originally the package had the `mongodb-log-writer` namespace. I think we have both namespaces in `devtools-shared`. Looks like the packages that was moved to the mono repo didn't add the `@mongodb-js/` prefix to their names. cc @addaleax

Reply:
probably worth preserving the old name just for the NPM history etc.